create McAfee NSM modeling rules (#27701)
demisto/content: Packs/McAfeeNSM/README.md

# McAfee NSM (Network Security Manager)
This pack includes Cortex XSIAM content.

## McAfee NSM Syslog configuration

> *McAfee NSM syslog event types:*

Reviewer: ```suggestion
McAfee NSM syslog event types:
```

* IPS Events
* Faults
* User Activity (audit logs)

*config Syslog IPS Events:*

> 1. on McAfee NSM go to Manager -> Setup -> Notification -> IPS Events -> syslog.

Reviewer: ```suggestion
1. In McAfee NSM, go to **Manager** > **Setup**> **Notification** > **IPS Events** > **syslog**.
```

> 2. click *YES* on "Enable Syslog Notification".

Reviewer: ```suggestion
2. In Enable Syslog Notification, click **YES .
```

> 3. bellow click on *+* and add Target server. 

Reviewer: ```suggestion
3. Click *+* and add the target server. 
```

>    * if you not configured target server alick add neer the "Target Server" and  fill the target server details.

Reviewer: ```suggestion
   If you do not have a configured target server, click **Add** near Target Server and  fill in the target server details.
```

   ![link](https://raw.githubusercontent.com/demisto/content/2063d324e6515a85b484705df5e4d153425e5110/Packs/McAfeeNSM/doc_imgs/nsm_add_target_server.png)

> 4. on Facility select "Log Alert (note 1)".

Reviewer: ```suggestion
4. In Facility, select **Log Alert (note 1)**.
```

> 5. Severity mapping - not touch, need to be:

Reviewer: ```suggestion
5. In Severity mapping, leave the configuration as it.  It should be:
```

 ![link](https://raw.githubusercontent.com/demisto/content/95eff3fe52a33695a10a76209cf8f2c4edbc185f/Packs/McAfeeNSM/doc_imgs/nsm_ips_severity_mapping.png)

> 6. click on the Checkbox to enable "Notify for all Alerts" and click save.

Reviewer: ```suggestion
6. Click the Notify for all Alerts checkbox.
7. Click **Save**.
```

> *config Syslog Faults Events:*

Reviewer: ```suggestion
Configure Syslog Faults Events:
```

> 1. on McAfee NSM go to Manager -> Setup -> Notification -> Faults -> syslog.

Reviewer: ```suggestion
1. In McAfee NSM, go to **Manager** > **Setup**> **Notification** > **Faults** > **syslog**.
```

> 2. click *YES* on "Enable Syslog Notification".

Reviewer: ```suggestion
2. In Enable Syslog Notification, click **YES**.
```

> 3. fill the fields "Server Name or IP Address" and "Port".

Reviewer: ```suggestion
3. Fill in the "Server Name or IP Address" and "Port" fields .
```

> 4. on *facilities* dropdown select "Security/authorization (code 4)".

Reviewer: ```suggestion
4. In the Facilities dropdown, select **Security/authorization (code 4)**.
```

> 5. severity mapping not touch, will be like:

Reviewer: ```suggestion
5. In Severity mapping, leave the configuration as it.  It should be:
```

    ![link](https://raw.githubusercontent.com/demisto/content/53399299a79f6d8323502c6489c02b87a8720a7b/Packs/McAfeeNSM/doc_imgs/nsm_faults_severity_mapping.png)

> 6. on *Forward Faults* dropdown select "informational and above".

Reviewer: ```suggestion
6. In the Forward Faults dropdown, select **informational and above**.
```

> 7. on *Message Preference* click on the checkbox "Syslog default" and click save.

Reviewer: ```suggestion
7. In Message Preference, click the Syslog default checkbox.
8. Click **Save**.
```

> *config Syslog User Activity (audit logs) Events:*

Reviewer: ```suggestion
Configure Syslog User Activity (audit logs) Events:
```

> 1. on McAfee NSM go to Manager -> Setup -> Notification -> User Activity -> syslog.

Reviewer: ```suggestion
1. In McAfee NSM, go to **Manager** > **Setup**> **Notification** > **User Activity** > **syslog**.
```

> 2. click *YES* on "Enable Syslog Notification".

Reviewer: ```suggestion
2. In Enable Syslog Notification, click **YES**.
```

> 3. fill *Server Name or IP Address* and *port*.  

Reviewer: ```suggestion
3. Fill in the "Server Name or IP Address" and "Port" fields .
```

> 4. select Protocol on *Protocol* dropdown.

Reviewer: ```suggestion
4. In the Protocol dropdown, select **Protocol**.
```

> 5. on *Facilities* dropdown select "Log Alert (note 1)".

Reviewer: ```suggestion
5. In the Facilities dropdown, select **Log Alert (note 1)**.
```

> 6. on severity mapping not touch, will be like:

Reviewer: ```suggestion
6. In Severity mapping, leave the configuration as it.  It should be:
```

    ![link](https://raw.githubusercontent.com/demisto/content/53399299a79f6d8323502c6489c02b87a8720a7b/Packs/McAfeeNSM/doc_imgs/nsm_audit_severity_mapping.png)

> 7. on *Forward audit* dropdown select "Allow All Auditlogs".

Reviewer: ```suggestion
7. In the Forward audit dropdown, select **Allow All Auditlogs**.
```

> 7. on *Message Preference* click on the checkbox "Syslog default" and click save.

Reviewer: ```suggestion
7. In Message Preference, click the Syslog default checkbox.
8. Click **Save**.
```

## Event Time configuration

> By Default on Fault and IPS events (syslog) not have event time, to add event time do the followin steps:

Reviewer: ```suggestion
By default, on Fault and IPS events (syslog) do not have an event time. To add an event time, perform the following:
```

*IPS Events*

> 1. on McAfee NSM go to Manager -> Setup -> Notification -> IPS Events -> syslog.

Reviewer: ```suggestion
1. In McAfee NSM, go to **Manager** > **Setup**> **Notification** > **IPS Events** > **syslog**.
```

> 2. choose target server and click on the pencil (edit)

Reviewer: ```suggestion
2. Choose the target server and click the pencil (edit).
```

> 3. on message part add in the end of the string " at  $IV_ATTACK_TIME$"

Reviewer: ```suggestion
3. In the message part, add in the end of the string " at  $IV_ATTACK_TIME$".
```

> 4. click save

Reviewer: ```suggestion
4. Click **Save**.
```

*Syslog Faults*

> 1. on McAfee NSM go to Manager -> Setup -> Notification -> Faults -> syslog.

Reviewer: ```suggestion
1. In McAfee NSM, go to **Manager** > **Setup**> **Notification** > **Faults** > **syslog**.
```

> 2. on message preference click edit and add to the end of the message " at  $IV_FAULT_TIME$"

Reviewer: ```suggestion
2. In Message Preference, click edit and add to the end of the message " at  $IV_FAULT_TIME$".
```

> 3. click save

Reviewer: ```suggestion
3. Click **Save**.
```
